nequip/data/AtomicData.py

```diff
@@ -249,6 +250,44 @@ def from_ase(cls, atoms, r_max, **kwargs):
             **add_fields,
         )
 
+    def to_ase(self) -> Union[List[ase.Atoms], ase.Atoms]:
+        """Build a (list of) ``ase.Atoms`` object(s) from an ``AtomicData`` object.
+
+        For each unique batch number associated with AtomicDataDict.BATCH_KEY,
+        an ``ase.Atoms`` object is created. If AtomicDataDict.BATCH_KEY does not
+        exist in self, a single ``ase.Atoms`` object is created.
```

Comment:
```suggestion
        For each unique batch number provided in ``AtomicDataDict.BATCH_KEY``,
        an ``ase.Atoms`` object is created. If ``AtomicDataDict.BATCH_KEY`` does not
        exist in self, a single ``ase.Atoms`` object is created.
```

The `` is a stupid formatting thing forced on us by Sphinx (our documentation framework); it makes things formatted as "code" (sort of... :D )